pydarn/plotting/fan.py

```diff
@@ -268,36 +262,41 @@ def plot_fov(cls, stid: str, dtime: dt.datetime, ax=None,
             thetas - theta polar coordinates
             rs - radius polar coordinates
         """
+        
+        #Set datetime object to current computer time if not given
+        if not date:
+            date = dt.datetime.now()     
+        
         # Get radar beam/gate locations
         beam_corners_aacgm_lats, beam_corners_aacgm_lons = \
-            radar_fov(stid, coords='aacgm', date=dtime)
+            radar_fov(stid, coords='aacgm', date=date)
         fan_shape = beam_corners_aacgm_lons.shape
 
         # Work out shift due in MLT
         beam_corners_mlts = np.zeros((fan_shape[0], fan_shape[1]))
         mltshift = beam_corners_aacgm_lons[0, 0] - \
-            (aacgmv2.convert_mlt(beam_corners_aacgm_lons[0, 0], dtime) * 15)
+            (aacgmv2.convert_mlt(beam_corners_aacgm_lons[0, 0], date) * 15)
         beam_corners_mlts = beam_corners_aacgm_lons - mltshift
 
         # Hold the beam positions
         thetas = np.radians(beam_corners_mlts)
-        rs = beam_corners_aacgm_lats
+        rs = beam_corners_aacgm_lats   
 
         # Setup plot
         # This may screw up references
         if ax is None:
-            ax = plt.axes(polar=True)
-            if beam_corners_aacgm_lats[0, 0] > 0:
-                ax.set_ylim(90, lowlat)
-                ax.set_yticks(np.arange(lowlat, 90, 10))
-            else:
-                ax.set_ylim(-90, -abs(lowlat))
-                ax.set_yticks(np.arange(-abs(lowlat), -90, -10))
-            ax.set_xticks([0, np.radians(45), np.radians(90), np.radians(135),
-                           np.radians(180), np.radians(225), np.radians(270),
-                           np.radians(315)])
-            ax.set_xticklabels(['00', '', '06', '', '12', '', '18', ''])
-            ax.set_theta_zero_location("S")
+        
+            # If a hemisphere kwarg wasn't passed in, determine the right hemisphere
```

Comment:
We can avoid this with Station id  https://github.com/SuperDARN/pydarn/blob/b58da297e685f92dacc2ca30771222f7becbad6f/pydarn/utils/superdarn_radars.py#L181